src/main/scala/com.socrata.tileserver/services/TileService.scala

```diff
@@ -77,7 +77,12 @@ case class TileService(client: CuratedServiceClient) extends SimpleResource {
   { resp: Response =>
     def createResponse(parsed: (JValue, Seq[FeatureJson])): HttpResponse = {
       val (jValue, features) = parsed
-      logger.debug(s"Underlying json: ${jValue.toString}")
+
+      // $COVERAGE-OFF$
+      if (logger.isDebugEnabled) {
```

Comment:
Oh, sorry, already merged, because I saw your comment about bad deps or
something.

On Wed, Apr 1, 2015 at 5:14 PM, Kaida Masaki <notifications@github.com>
wrote:

> In src/main/scala/com.socrata.tileserver/services/TileService.scala
> <https://github.com/socrata/tileserver/pull/19#discussion_r27624359>:
>
> > @@ -77,7 +77,12 @@ case class TileService(client: CuratedServiceClient) extends SimpleResource {
> >    { resp: Response =>
> >      def createResponse(parsed: (JValue, Seq[FeatureJson])): HttpResponse = {
> >        val (jValue, features) = parsed
> > -      logger.debug(s"Underlying json: ${jValue.toString}")
> > +
> > +      // $COVERAGE-OFF$
> > +      if (logger.isDebugEnabled) {
>
> I'm actually using logback. That does look simpler.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/socrata/tileserver/pull/19/files#r27624359>.
>



-- 
Evan Chan   |   Principal Engineer, Socrata     |    @evanfchan
*Socrata <http://www.socrata.com/> *| The Data Experience Company™
*Socrata isn't about worldly success, but about changing the world
successfully. This is why I'm excited to be here.*
*What does Socrata do? <https://www.youtube.com/watch?v=yyz86MlkriQ> *(watch
our 2 min video)